src/models/products.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @name state model
+ * @author khushbuw
+ */
+const mongoose = require('mongoose');
+const country = require('./categories');
+var Schema = mongoose.Schema;
+
+const productSchema = new mongoose.Schema({
+    category_id: {
+        type: Schema.Types.ObjectId,
+        ref: 'categories',
+        required: true,
+        trim: true,
+    },
```

Comment:
Please check thoroughly and refactor this code carefully. Please reach out to me in case of any confusion.